sssd_test_framework/roles/ad.py

```diff
@@ -800,6 +806,10 @@ def modify(
         :type shell: str | DeleteAttribute | None, optional
         :param email: Email address, defaults to None
         :type email: str | DeleteAttribute | None, optional
+        :param given_name: Given name of user, defaults to None
```

Comment:
Variable names look strange. I'd prefer 

givenname instead given_name and surname instead of sur_name

It's just because they point to an LDAP attribute. It's a small detail; reconsider it. Otherwise, it looks good. Thanks!

Reply:
ok.  fixed.  Not sure why I added the underscore in the names there.